pypy/module/_multiprocessing/interp_semaphore.py

```diff
@@ -202,8 +208,9 @@ def sem_getvalue(sem):
 
     def gettimeofday():
         now = lltype.malloc(TIMEVALP.TO, 1, flavor='raw')
+        void = lltype.nullptr(rffi.VOIDP.TO)
         try:
-            res = _gettimeofday(now, None)
+            res = _gettimeofday(now, void)
```

Comment:
This should use c_gettimeofday from rpython/rlib/rtime

Reply:
This also brings up 64-bit time
Glibc doc: https://sourceware.org/glibc/wiki/Y2038ProofnessDesign

Debian uses 64-bit time on all 32-bit architectures except i386 (i386 mostly exists for compatibility for older binaries)